test/vim_test.js

```diff
@@ -4949,7 +4972,7 @@ testVim('ex_map_key2ex', function(cm, vim, helpers) {
   eq(actualCm, cm);
 });
 testVim('ex_map_key2key_visual_api', function(cm, vim, helpers) {
-  CodeMirror.Vim.map('b', ':w', 'visual');
+  CodeMirror.Vim.map('b', ':w<CR>', 'visual');
```

Comment:
This behavior of not requiring <cr> at the end did not match with vim. Not sure if it was an intentional change or simply a limitation of the previous implementation.